lib/ui/src/keyboard/keyCodes.ts

```diff
@@ -354,15 +354,15 @@ const userSettingsGeneralMap = new KeyCodeStrMap();
 })();
 
 // tslint:disable-next-line:no-namespace
-export namespace KeyCodeUtils {
-  export function toString(keyCode: KeyCode): string {
-    return uiMap.keyCodeToStr(keyCode);
-  }
+// export namespace KeyCodeUtils {
```

Comment:
can this be removed?

Reply:
Just talked to @ndelangen 
It came in by merging next into this branch - I'll leave it as is because it might be needed by the code owner